macro/PackageHeader.py

```diff
@@ -7,14 +7,11 @@
 generates_headings = True
 dependencies = []
 
-if 'boxturtle' in distro_names:
-    distro_names.remove('boxturtle')
-if 'cturtle' in distro_names:
-    distro_names.remove('cturtle')
-if 'diamondback' in distro_names:
-    distro_names.remove('diamondback')
-if 'unstable' in distro_names:
-    distro_names.remove('unstable')
+hidden_distros = ['boxturtle', 'cturtle', 'diamondback', 'electric', 'fuerte', 'groovy', 'hydro', 'unstable']
+
+for hidden_distro in hidden_distros if hidden_distro in distro_names:
+    distro_names.remove(hidden_distro)
```

Comment:
I would suggest using list comprehension for this:

`distro_names = [d for d in distro_names if d not in hidden_distros]`